### ISSUE TITLE
Using range argument with read_xlsx

### ISSUE THREAD

**Issue by maloherry** [JanMarvin/openxlsx2#635]
First, thanks for this great plugin!
I guess this is more of a feature request: would it be possible to use a `range` argument in the read_xlsx function, as it is in packages like `readxl`, so that we can write something like this:
`read_xlsx("file.xlsx","sheet", range = "A1:C40")`

**Comment by JanMarvin** [comment]
Hi @maloherry , you can use `dims` for this. It's use is documented in `wb_to_df()`.

**Comment by maloherry** [comment]
Oh great thanks! (Maybe it would be relevant to add it to the `read_xlsx` [documentation](https://janmarvin.github.io/openxlsx2/reference/read_xlsx.html) then)

**Comment by JanMarvin** [comment]
> Oh great thanks! (Maybe it would be relevant to add it to the `read_xlsx` [documentation](https://janmarvin.github.io/openxlsx2/reference/read_xlsx.html) then)

Yeah, we do our best, but documentation is not really our strong suite :smile: I guess we could simply warp `read_xlsx()`, `wb_read()` and `wb_to_df()` in a single man page, but there is always something to improve. Feel free to send in a pull request :)

**Comment by olivroy** [comment]
Hi @maloherry, the docs were improved for the 0.8 release. Feel free to take a look. https://janmarvin.github.io/openxlsx2/reference/wb_to_df.html
